storage/client/blob/src/test/java/com/azure/storage/blob/APISpec.groovy

```diff
@@ -42,8 +40,6 @@ class APISpec extends Specification {
 
     static final ByteBuffer defaultData = ByteBuffer.wrap(defaultText.getBytes(StandardCharsets.UTF_8))
 
-    static final Flux<ByteBuffer> defaultFlux = Flux.just(defaultData)
```

Comment:
Are the items removed in this file just not being used anywhere?

Reply:
Correct, removed as nothing was referencing this.